salesagility/SuiteCRM
# Fixed #3906 - Pre-filled RELATE TO field in compose email pop up window isn't working for CASE in 7.9.x

Task: SCRM-658

## Description



When composing an email from "Cases"->"Detail-View"->"Activities-Sub-panel"->"Compose Email", the related to drop-down and field name do not populate.

## Motivation and Context

This change ensures that when a module like "Cases" does not have an email1 field that the composer will still relate the email.

Issue reference: #3906

## How To Test This

1. Go into "Cases"->"Detail-View".
2. Enter the sub-panel "Activities"->"Actions Drop-down"->"Compose Email".
3. Check that the email popup composer has a blank "To" field and that the email is related to the case.
4. Check that the same functionality is present in other modules like "Accounts" but with the "To" field populated.

## Types of changes


### Final checklist
